components/style/properties/gecko.mako.rs

```diff
@@ -790,6 +790,11 @@ fn static_assert() {
         Au(self.gecko.mSize)
     }
 
+    <% kerning_keyword = Keyword("font-kerning", "auto normal none",
+                                 gecko_constant_prefix='NS_FONT_KERNING') %>
+
+    ${impl_keyword('font_kerning', 'mFont.kerning', kerning_keyword, need_clone=False, cast_type='u8')}
```

Comment:
Ah, OK. Fixed in https://github.com/servo/servo/pull/13751/commits/b4e91f648f0d7a0b46f753f70df02f2d22e9fa8d